docs: add conditional deployment use case to readme (#289)
scolladon/sfdx-git-delta: README.md

 */
```

> ### Condition a deployment if package.xml and destructiveChange are empty

Reviewer: **suggestion:** simpler form

```suggestion
### Condition deployment on package.xml and destructiveChange content
```

Reviewer 2: ```suggestion
### Condition deployment on package.xml and destructiveChange content:
```

> Since we can add files to .sgdignore, [some commits on those files can lead to empty package.xml and destructiveChanges.xml and cause deployment errors.](https://github.com/scolladon/sfdx-git-delta/issues/249)
> 
> To adress this requirement, you can test files content and condition your command to run:

Reviewer: **suggestion:** I propose this sentence (we use [hemingwayApp](https://hemingwayapp.com/) to create doc)

```suggestion
SGD does not always generate content in the package.xml (or destructiveChanges.xml). Sometimes the commit range contains changes only within files to ignore (using .sgdignore and `--i` parameter).
[Deploying empty package.xml can lead to deployment errors.](https://github.com/scolladon/sfdx-git-delta/issues/249)
To avoid starting a failing deployment, test files content before execution:
```
